### ISSUE TITLE
Issue with Getting Started

### ISSUE THREAD

**Issue by mikebrules** [inuitcss/getting-started#13]
Hi, cracking README, thanks; great detail. There is an issue with the Getting started section (when you are new to it anyway) - 

https://github.com/inuitcss/getting-started#as-quick-as-possible

If you install the starter kit, the modules listed as needed to be imported are not under the projectsa node_modules - they are under inuit-starterkit's node_modules. Just slightly confusing.

Thanks

**Comment by csshugs** [comment]
@mikebrules 
The idea is (as I understand it) to start from scratch, i.e. going into an empty folder and from there run:

```
$ npm install --save inuit-starter-kit
```

After doing so, there should be a `node_modules` folder containing the modules.

**Comment by csshugs** [comment]
When you did install the starter kit via a custom `package.json` in your project directory, e.g.:

```
{
  "name": "project",
  "version": "0.1.0",
  "author": "me",
  "dependencies": {
    "inuit-starter-kit": "0.2.9"
  }
}
```

...and then did:

```
$ npm install
```

If you did it this way, it might have been an old version of npm, which caused the modules to nest, which means: Your project has one dependency (`inuit-starter-kit`). And this dependency has other dependencies (defined in `inuit-starter-kit`). So your `/node_modules` folder looks like this:

```
/node_modules
  /inuit-starter-kit
    /inuit-defaults
    /inuit-functions
    /inuit-mixins
    /inuit-box-sizing
    /inuit-normalize
    /inuit-page
```

Whereas in the new version, it'll look like this*:

```
/node_modules
  /inuit-starter-kit
  /inuit-defaults
  /inuit-functions
  /inuit-mixins
  /inuit-box-sizing
  /inuit-normalize
  /inuit-page
```

Anyway, the starter kit is meant to be used as the initial starting point of your project as described above.

---

\* I picked this up somewhere. I'm not 100% sure if I got this right. At least it looks like that on my machine when I run `npm install`.
